src/Evolution/Systems/Cce/InitializeCce.hpp

```diff
@@ -32,19 +88,31 @@ namespace Cce {
  * B &= - \frac{1}{2} R^3 (J|_{r = R} + R \partial_r J|_{r = R})
  * \f}
  */
-template <template <typename> class BoundaryPrefix>
-struct InitializeJ {
-  using boundary_tags = tmpl::list<BoundaryPrefix<Tags::BondiJ>,
-                                   BoundaryPrefix<Tags::Dr<Tags::BondiJ>>,
-                                   BoundaryPrefix<Tags::BondiR>>;
+struct InitializeJInverseCubic : public InitializeJ {
+  using options = tmpl::list<>;
+  static constexpr OptionString help = {
+      "Initialization process where J is set to a simple Ansatz with a A/r + "
+      "B/r^3 piece such that it is smooth with the Cauchy data at the "
+      "worldtube"};
```

Comment:
Other classes add explicit line breaks `\n` in their help strings, I'm not sure right now if that's needed or not though